pkg/statistics/handle/ddl/ddl.go

```diff
@@ -183,7 +186,8 @@ func (h *ddlHandlerImpl) HandleDDLEvent(s *ddlutil.SchemaChangeEvent) error {
 	case model.ActionFlashbackCluster:
 		return h.statsWriter.UpdateStatsVersion()
 	default:
-		return errors.Trace(errors.Errorf("unhandled DDL type %v", s.GetType()))
+		intest.Assert(false)
+		logutil.StatsLogger().Error("Unhandled schema change event", zap.Stringer("type", s.GetType()))
```

Comment:
Do we need to print the whole event here?

Reply:
What’s the purpose of printing the whole event in log?

Reply:
I just thought that the purpose of this default branch is to remind developer there is an unhandled event, any other use?  🤔

Reply:
I guess it would be easier to figure out what information is in it. But I am okay with it if you only want to print the type here.

Reply:
PTAL again https://github.com/pingcap/tidb/pull/55973/commits/8b171a40c93a5abb2d25ff43e8a90d96c1a9c4a1